spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -17,37 +17,144 @@
 // limitations under the License.
 // Contributors:
 // Steve Fontes
-[[architecture]]
 
 == Architecture
 
-This specification defines an easy way to instrument an application to take advantage of an opentracing.io Tracer implementation.
+This specification defines an easy way to allow an application running in a
+microprofile.io container to take advantage of distributed tracing by using an
+opentracing.io Tracer implementation.
 
+=== Rationale
 
-=== Rational
+Distributed tracing allows you to trace the flow of a request across service
+boundaries. This is particularly important in a microservices environment where
+a request typically flows through multiple services. To accomplish distributed
+tracing, each service must be instrumented to log messages with a correlation id
+that may have been propagated from an upstream service. A common companion to
+distributed trace logging is a service where the distributed trace records can
+be stored.
+([Examples](http://opentracing.io/documentation/pages/supported-tracers.html)).
+The storage service for distributed trace records can provide features to view
+the cross service trace records associated with particular request flows.
 
-Distributed tracing allows you to trace the flow of a request across service boundaries.
-This is particularly important in a microservices environment where a request typically flows through multiple services.
-To accomplish distributed tracing, each service must be instrumented to log messages with a correlation id that may have been propagated from an upstream service.
-A common companion to distributed trace logging is a service where the distributed trace records can be stored. ([Examples](http://opentracing.io/documentation/pages/supported-tracers.html)).
-The storage service for distributed trace records can provide features to view the cross service trace records associated with particular request flows.
+Opentracing.io is an API specification for working with a distributed tracing
+system. Opentracing.io defines a Span as the conceptual artifact that is
+propagated between services. The Span information can also be communicated to a
+central server for storage.
 
-Opentracing.io is an API specification for working with a distributed tracing system. Opentacing.io defines a Span as the conceptual artifact that is propagated between services. The Span information can also be communicated to a central server for storage.
+The microprofile.io Opentracing proposal will describe how a developer can
```

Comment:
s/Openracing/OpenTracing, possibly applies to more places

Reply:
done